src/image-target/aframe.js

```diff
@@ -171,13 +171,43 @@ AFRAME.registerSystem('mindar-image-system', {
       }
     }
 
+    this._resize();
+    window.addEventListener('resize', this._resize.bind(this));
+
     await this.controller.dummyRun(this.video);
     this.el.emit("arReady");
     this.ui.hideLoading();
     this.ui.showScanning();
 
     this.controller.processVideo(this.video);
   },
+
+  _resize: function() {
```

Comment:
In that case, we should use this _resize method at start as well to avoid duplicating code.

Reply:
I tried to fix this in the in my rewrite